src/components/Heading-h2/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const HeadingH2 = ({ primary, secondary }) => {
```

Comment:
please change component name (and file name) to (Heading), it is not a good practice to rely on any tag inside the component in component name, for example, in the future if we decided that all headers will be `h3` or even a `p`, the component name will be misleading.